crates/swc_plugin_import/src/lib.rs

```diff
@@ -473,7 +473,7 @@ where
     let s: &str = self.as_ref();
     let mut output = String::with_capacity(s.len());
 
-    s.chars().enumerate().for_each(|(idx, c)| {
```

Comment:
Using `chars().enumerate()` is fine here, cause the function looks just wants to do something different for the first char.